draft-jones-oauth-rfc7523bis.xml

```diff
@@ -265,20 +267,25 @@
 	  </t>
 	  <t>
 	    The JWT MUST contain an <spanx style="verb">aud</spanx>
-	    (audience) claim containing a value that
-	    identifies the authorization server as an intended audience.
-	    The token endpoint URL of the authorization server MAY be used as a
-	    value for an <spanx style="verb">aud</spanx> element to identify
-	    the authorization server as an intended audience of the JWT.
+	    (audience) claim containing
+	    the issuer identifier <xref target="RFC8414"/>
+	    of the authorization server as its sole value.
+	    The authorization server MUST have an issuer identifier
+	    to be used with this specification.
+	    Unlike the <spanx style="verb">aud</spanx> value specified
+	    in <xref target="RFC7523"/>, there MUST be no value other than
+	    the issuer identifier of the intended authorization server
+	    used as the audience of the JWT;
+	    this includes that the token endpoint URL of the authorization server
+	    MUST NOT be used as an audience value.
+	    It is RECOMMENDED that the <spanx style="verb">aud</spanx> claim value
```

Comment:
Given this is going to be a breaking change, can i recommend that we remove the support for both  a string and array with a single string in it. For simplicity (which is its own security) and compute cycles and implementation cmplexity by both AS as and RPs wouldn't it make more sense to drop the ARRAY support with a single value in it, for just supporting and requiring a single string. my 2p